Makefile

```diff
@@ -48,6 +49,11 @@ $(IMAGE): $(LXC) profile
 # Set up kubectl
 	$(LXC) exec $(INSTANCE) -- snap install kubectl --classic
 
+# Set up postdeploy options
+	$(LXC) exec $(INSTANCE) -- mkdir -p /home/ubuntu/bin
+	$(LXC) exec $(INSTANCE) -- sh -c "echo 'PATH=$$PATH:/home/ubuntu/bin' >> $(USERHOME)/.bashrc"
```

Comment:
Shoudl we put /home/ubuntu/bin at the beginning of the path, so that it can optionally be used to override global commands in case it's required? YAGNI? [shrug]

Reply:
Leaving it like this means path is the same in termserver and limited termserver. In lshell, env_path always appends.